README.md

```diff
@@ -107,7 +107,7 @@ jobs:
 
 | Name                  | Description                                                                                                                    | Default                  |
 |-----------------------|--------------------------------------------------------------------------------------------------------------------------------|--------------------------|
-| `docker-registry`     | Docker Registry                                                                                                                | `registry.staffbase.com` |
+| `docker-registry`     | Docker Registry                                                                                                                | `staffbase.jfrog.io`     |
```

Comment:
Do we relay want to publish our internaly used registry url?
Sure, its easy to guess but still...

Reply:
I would set this as a mandatory variable and not set our organization default for this public github-action. We should set these environment variables by our own.